app/konnect/centralized-consumers.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Centralized Consumers
+alpha: true
```

Comment:
```suggestion
```
This is going to GA. See https://kongstrong.slack.com/archives/C06FJ632KJS/p1741905058701429?thread_ts=1741903435.247969&cid=C06FJ632KJS